### PR TITLE
Feat/use UI provider

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

**Comment by zachferland** [comment]
@PaulLeCam going to plan to merge anyways (with ci fail), think it make sense to keeping move here, tests pass locally for me, but are still sometimes on unstable, think it continuing on stories this sprint and refactor will clear it more up here

File: apps/iframe/src/index.ts [hunk 1]
```diff
@@ -3,6 +3,9 @@ import template from './html/template'
 
 import { ConnectService } from './connectService'
 import type { UserRequestHandler } from './types'
+import { UIProvider, UIProviderHandlers } from '../../../packages/ui-provider/src/index'
+import { BlobOptions } from 'buffer'
```
Comment (PaulLeCam) [inline apps/iframe/src/index.ts:7]: Is this import intentional?

File: apps/iframe/src/index.ts [hunk 2]
```diff
@@ -3,6 +3,9 @@ import template from './html/template'
...
+import { create } from 'eslint/lib/rules/*'
```
Comment (PaulLeCam) [inline apps/iframe/src/index.ts:8]: Probably not intentional

File: apps/iframe/package.json [hunk 3]
```diff
@@ -13,14 +13,15 @@
     "start:testnet": "CERAMIC_API=https://ceramic-clay.3boxlabs.com npm run start"
   },
   "dependencies": {
-    "3id-did-provider": "^1.0.2",
     "@3id/common": "^0.1.3",
     "@3id/connect-display": "^0.1.5",
     "@3id/manager": "^0.1.6",
+    "@3id/ui-provider": "file:../../packages/ui-provider",
```
Comment (PaulLeCam) [inline apps/iframe/package.json:19]: That shouldn't be needed if the monorepo is working, have you run `yarn` after adding the package?

File: apps/iframe/src/index.ts [hunk 4]
```diff
@@ -3,6 +3,9 @@ import template from './html/template'
 
 import { ConnectService } from './connectService'
 import type { UserRequestHandler } from './types'
+import { UIProvider, UIProviderHandlers } from '../../../packages/ui-provider/src/index'
```
Comment (PaulLeCam) [inline apps/iframe/src/index.ts:6]: `... from '@3id/ui-provider'` no?

**Review by PaulLeCam — APPROVED** [review]

**Comment by PaulLeCam** [comment]
@zachferland yes I agree better keep moving with this, the logic looks good I think it will just need some cleanup with the imports.

**Comment by zachferland** [comment]
ah yup, forgot to clean those up before, thanks!

**Comment by zachferland** [comment]
lol and the test pass now, could have been part of the issue i guess, either way will aim for more test stability after, dont think it would reliably pass

**Pull request merged**